pkg/options/options.go

```diff
@@ -76,7 +76,7 @@ var DBImage = "centos/mongodb-36-centos7"
 
 // DBPostgresImage is the default postgres db image url
 // currently it can not be overridden.
-var DBPostgresImage = "centos/postgresql-12-centos7"
+var DBPostgresImage = "quay.io/sclorg/postgresql-15-c8s"
```

Comment:
we need to check with Boris that the downstream image is compatible with this one (from ENVs perspective, paths, etc.), it was the case for postgres12, but we need to make sure it is for 15

Reply:
I asked him, let's see what will be his reply

Reply:
@dannyzaken I discussed both with Boris and with @liranmauda and I think we should use this image for the upstream as it makes more sense: `[sclorgpostgresql-15-c9s](https://quay.io/repository/sclorg/postgresql-15-c9s)` (centos 9 like we will use RHEL 9) 
I will also re-test the happy path with the downstream image to validate it works like we expect